networking/hardware_networks/uninstalling-sriov-operator.adoc

```diff
@@ -0,0 +1,57 @@
+[id="uninstalling-sriov-operator"]
+= Uninstalling the SR-IOV Network Operator
+include::modules/common-attributes.adoc[]
+:context: uninstalling-sr-iov-operator
+
+toc::[]
+
+To uninstall the SR-IOV Network Operator, you must first delete any running SR-IOV workloads. You must also delete the webhooks after uninstalling the operator.
+
+.Prerequisites
+
+* You must have access to an {product-title} cluster using an account with `cluster-admin` permissions.
+* You have the SR-IOV Network Operator installed.
+
+.Procedure
+
+. Delete all SR-IOV custom resources (CRs):
++
+[source,terminal]
+----
+$ oc delete sriovnetwork -n openshift-sriov-network-operator --all
+$ oc delete sriovnetworknodepolicy -n openshift-sriov-network-operator --all
+$ oc delete sriovibnetwork -n openshift-sriov-network-operator --all
+----
+
+. Follow the instructions on xref:../../operators/admin/olm-deleting-operators-from-cluster.adoc#olm-deleting-operators-from-a-cluster[Deleting operators from a cluster] to remove the SR-IOV Network Operator from your cluster.
```

Comment:
You shouldn't link to steps in another context like this. You should include them like in the [assembly template](https://raw.githubusercontent.com/redhat-documentation/modular-docs/master/modular-docs-manual/files/TEMPLATE_ASSEMBLY_a-collection-of-modules.adoc).

Reply:
I wonder how the page would look like by adding the include in the middle of a procedure step. I'm thinking I'll have to split the procedure into multiple procedure modules, similar to https://docs.openshift.com/container-platform/4.9/serverless/install/removing-openshift-serverless.html. Please share a change suggestion if you have one.

Also, as a documentation user, I find it out of place to include the whole OLM deleting operators module in the SR-IOV uninstall page. The instructions will be generic, referencing the Jaeger Operator as an example.

Waiting for further clarification.

Reply:
It might be relevant to note that there are 5 occurrences of the same xref pattern in the openshift-docs repository:

```
[openshift-docs]$ git grep -l "xref:../../operators/admin/olm-deleting-operators" 
cicd/gitops/uninstalling-openshift-gitops.adoc
cicd/pipelines/uninstalling-pipelines.adoc
distr_tracing/distr_tracing_install/distr-tracing-removing.adoc
jaeger/jaeger_install/rhbjaeger-removing.adoc
operators/admin/olm-status.adoc
virt/backup_restore/virt-installing-configuring-oadp.adoc
```